Makefile

```diff
@@ -1,7 +1,7 @@
 # Makefile
-IMAGE_NAME = python:3.10-bookworm
-CONTAINER_NAME = appimage-builder-container-aarch64
-APPIMAGE_NAME = viam-camera-oak-d--aarch64.AppImage
+IMAGE_NAME = appimage-builder-image
+CONTAINER_NAME = appimage-builder-container
+AARCH64_APPIMAGE_NAME = viam-camera-oak-d--aarch64.AppImage
```

Comment:
It's double hyphenated due to how appimage-builder works. It requires a version number in `AppImageBuilder.yml`. I feed it an empty string though because I am not sure how to handle versioning in the make command. I figured this was okay since version management is handled entirely by Viam i.e. the users never see that appimage file themelves.